bioimageio_core_version: 0.5.3.post2
bioimageio_spec_version: 0.4.5.post11
rdf_sha256: 4cbd435f08e6a8b846105b33d642d29028d01ad2feec7b3e511333cc8b2e406d
status: passed
tests:
  bioimageio:
  - bioimageio_spec_version: 0.4.5post11
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.5 with update to latest
      format version
    nested_errors: {}
    status: passed
    traceback: null
    warnings:
      tags: 'Missing tags for categories: [{''modality'': [''electron-microscopy'',
        ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'', ''transmission-light-microscopy'',
        ''super-resolution-microscopy'', ''x-ray-microscopy'', ''force-microscopy'',
        ''high-content-imaging'', ''whole-slide-imaging'', ''other'']}, {''dims'':
        [''2d'', ''3d'', ''2d-t'', ''3d-t'']}, {''framework'': [''tensorflow'', ''pytorch'',
        ''tensorflow.js'']}, {''software'': [''ilastik'', ''imagej'', ''fiji'', ''imjoy'',
        ''deepimagej'', ''n2v'']}, {''network'': [''unet'', ''densenet'', ''resnet'',
        ''inception'', ''shufflenet'']}, {''task'': [''semantic-segmentation'', ''instance-segmentation'',
        ''object-detection'', ''image-classification'', ''denoising'', ''image-restoration'',
        ''image-reconstruction'', ''in-silico-labeling'']}]'
      weights: {onnx: missing 'opset_version'}
  - bioimageio_spec_version: 0.4.5post11
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.5
    nested_errors: {}
    status: passed
    traceback: null
    warnings:
      tags: 'Missing tags for categories: [{''modality'': [''electron-microscopy'',
        ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'', ''transmission-light-microscopy'',
        ''super-resolution-microscopy'', ''x-ray-microscopy'', ''force-microscopy'',
        ''high-content-imaging'', ''whole-slide-imaging'', ''other'']}, {''dims'':
        [''2d'', ''3d'', ''2d-t'', ''3d-t'']}, {''framework'': [''tensorflow'', ''pytorch'',
        ''tensorflow.js'']}, {''software'': [''ilastik'', ''imagej'', ''fiji'', ''imjoy'',
        ''deepimagej'', ''n2v'']}, {''network'': [''unet'', ''densenet'', ''resnet'',
        ''inception'', ''shufflenet'']}, {''task'': [''semantic-segmentation'', ''instance-segmentation'',
        ''object-detection'', ''image-classification'', ''denoising'', ''image-restoration'',
        ''image-reconstruction'', ''in-silico-labeling'']}]'
      weights: {onnx: missing 'opset_version'}
  - {bioimageio_core_version: 0.5.3post2, bioimageio_spec_version: 0.4.5post11, error: null,
    name: reproduced test outputs from test inputs, status: passed, traceback: null}
  deepimagej:
  - bioimageio_core_version: 0.5.3post2
    bioimageio_spec_version: 0.4.5post7
    error: 'unsupported operand type(s) for +: ''PosixPath'' and ''str'''
    name: reproduce test outputs with deepimagej 2.1.13 using onnx weights
    reason: error
    status: failed
    traceback: ["  File \"/home/runner/work/models/models/scripts/check_rdf_and_test_model.py\",
        line 122, in test_resource\n    error = (error or \"\") + download_deepimagej_model(fiji_path,
        rdf_path)\n", "  File \"/home/runner/work/models/models/scripts/utils.py\",
        line 27, in download_deepimagej_model\n    os.makedirs(fiji_dir + \"//models\")\n"]
  ilastik:
  - {error: Missing/invalid weight formats for 10.5281/zenodo.5910163/5942853, name: reproduce
      test outputs with ilastik 1.4.0b26, reason: error, status: failed}
  - {error: Missing/invalid weight formats for 10.5281/zenodo.5910163/5942853, name: reproduce
      test outputs with ilastik 1.4.0b27, reason: error, status: failed}
  - bioimageio_core_version: 0.5.3post2
    bioimageio_spec_version: 0.4.5post6
    error: No module named 'onnxruntime'
    name: reproduce test outputs with ilastik 1.4.0b27 using onnx weights
    reason: error
    status: failed
    traceback: ["  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 148, in test_resource\n    bioimageio_model=model, devices=devices, weight_format=weight_format\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 196, in create_prediction_pipeline\n    bioimageio_model=bioimageio_model,
        devices=devices, weight_format=weight_format\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
        line 113, in create_model_adapter\n    adapter_cls = _get_model_adapter(weight)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
        line 138, in _get_model_adapter\n    from ._onnx_model_adapter import ONNXModelAdapter\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_onnx_model_adapter.py\",
        line 5, in <module>\n    import onnxruntime as rt\n"]
  - bioimageio_core_version: 0.5.1
    bioimageio_spec_version: 0.4.5post3
    error: No module named 'onnxruntime'
    name: reproduce test outputs with ilastik 1.4.0b26 using onnx weights
    reason: error
    status: failed
    traceback: ["  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 148, in test_resource\n    bioimageio_model=model, devices=devices, weight_format=weight_format\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 159, in create_prediction_pipeline\n    bioimageio_model=bioimageio_model,
        devices=devices, weight_format=weight_format\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
        line 113, in create_model_adapter\n    adapter_cls = _get_model_adapter(weight)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
        line 138, in _get_model_adapter\n    from ._onnx_model_adapter import ONNXModelAdapter\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_onnx_model_adapter.py\",
        line 5, in <module>\n    import onnxruntime as rt\n"]
